en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-260-to-300.md

```diff
@@ -37,314 +320,1598 @@ Follow the instructions below to move all the existing API Manager configuration
+        PT_SCOPE_ID INTEGER NOT NULL,
+        PRIMARY KEY (ID),
+        FOREIGN KEY (PT_RESOURCE_ID) REFERENCES IDN_UMA_PT_RESOURCE (ID) ON DELETE CASCADE,
+        FOREIGN KEY (PT_SCOPE_ID) REFERENCES IDN_UMA_RESOURCE_SCOPE (ID) ON DELETE CASCADE
+        )
+        /
+
+        CREATE SEQUENCE IDN_UMA_PT_RESOURCE_SCOPE_SEQ START WITH 1 INCREMENT BY 1 NOCACHE
+        /
+
+        CREATE OR REPLACE TRIGGER IDN_UMA_PT_RESOURCE_SCOPE_TRIG
+        BEFORE INSERT
+        ON IDN_UMA_PT_RESOURCE_SCOPE
+        REFERENCING NEW AS NEW
+        FOR EACH ROW
+        BEGIN
+            SELECT IDN_UMA_PT_RESOURCE_SCOPE_SEQ.nextval INTO :NEW.ID FROM dual;
+        END;
+        /
+        ```
+        
+        ```tab="PostgreSQL"
+        CREATE SEQUENCE AM_SYSTEM_APP_SEQUENCE START WITH 1 INCREMENT BY 1 ;
+        CREATE TABLE IF NOT EXISTS AM_SYSTEM_APPS (
+                    ID INTEGER DEFAULT nextval('am_system_app_sequence'),
+                    NAME VARCHAR(50) NOT NULL,
+                    CONSUMER_KEY VARCHAR(512) NOT NULL,
+                    CONSUMER_SECRET VARCHAR(512) NOT NULL,
+                    CREATED_TIME TIMESTAMP(6) DEFAULT CURRENT_TIMESTAMP(6),
+                    PRIMARY KEY (ID)
+        );
+
+        CREATE TABLE IF NOT EXISTS AM_API_CLIENT_CERTIFICATE (
+        TENANT_ID INTEGER NOT NULL,
+        ALIAS VARCHAR(45) NOT NULL,
+        API_ID INTEGER NOT NULL,
+        CERTIFICATE BYTEA NOT NULL,
+        REMOVED BOOLEAN NOT NULL DEFAULT 0,
+        TIER_NAME VARCHAR (512),
+        FOREIGN KEY (API_ID) REFERENCES AM_API (API_ID) ON DELETE CASCADE ON UPDATE CASCADE,
+        PRIMARY KEY (ALIAS,TENANT_ID, REMOVED)
+        );
+
+        ALTER TABLE AM_POLICY_SUBSCRIPTION ADD MONETIZATION_PLAN VARCHAR(25) NULL DEFAULT NULL,
+        ADD FIXED_RATE VARCHAR(15) NULL DEFAULT NULL, 
+        ADD BILLING_CYCLE VARCHAR(15) NULL DEFAULT NULL, 
+        ADD PRICE_PER_REQUEST VARCHAR(15) NULL DEFAULT NULL, 
+        ADD CURRENCY VARCHAR(15) NULL DEFAULT NULL;
+
+        CREATE TABLE IF NOT EXISTS AM_MONETIZATION_USAGE_PUBLISHER (
+            ID VARCHAR(100) NOT NULL,
+            STATE VARCHAR(50) NOT NULL,
+            STATUS VARCHAR(50) NOT NULL,
+            STARTED_TIME VARCHAR(50) NOT NULL,
+            PUBLISHED_TIME VARCHAR(50) NOT NULL,
+            PRIMARY KEY(ID)
+        );
+
+        CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+        ALTER TABLE AM_API_COMMENTS
+        DROP COLUMN COMMENT_ID,
+        ADD COLUMN COMMENT_ID VARCHAR(255) NOT NULL DEFAULT uuid_generate_v1(),
+        ADD PRIMARY KEY (COMMENT_ID);
+
+        ALTER TABLE AM_API_RATINGS
+        DROP COLUMN RATING_ID,
+        ADD COLUMN RATING_ID VARCHAR(255) NOT NULL DEFAULT uuid_generate_v1(),
+        ADD PRIMARY KEY (RATING_ID);
+
+        CREATE TABLE IF NOT EXISTS AM_NOTIFICATION_SUBSCRIBER (
+            UUID VARCHAR(255),
+            CATEGORY VARCHAR(255),
+            NOTIFICATION_METHOD VARCHAR(255),
+         
```

Comment:
This URL needs to be updated

Reply:
The URL?

Reply:
Referred this URL - https://docs.wso2.com/display/CLUSTER44x/Configuring+SVN-Based+Deployment+Synchronizer

Reply:
For now, removed this content from the page. Will add it back after having an internal discussion.